content/en/docs/started/workstation/getting-started-linux.md

```diff
@@ -42,32 +40,33 @@ The only following applications are required to use MiniKF:
 The full set of instructions are available on the
 [MiniKF getting started](/docs/started/workstation/getting-started-minikf/) page.
 
-#### Kind
+### Kind
 
 [kind](https://kind.sigs.k8s.io/) is a tool for running local Kubernetes clusters using Docker container "nodes".
 kind was primarily designed for testing Kubernetes itself, but may be used for local development or CI.
 
 The full set of instructions are available on the
 [kind getting started](/docs/other-guides/virtual-dev/getting-started-kind/) page.
 
-### Linux appliance
+### Multipass
 
-A Linux appliance is a virtual machine that holds the linux operating system. From there
-you have complete choice over Kubernetes and Kubeflow, which offers the greatest degree
-of flexibility. You only need to install a single application to follow this path:
+[Multipass](https://multipass.run/) creates a fast Linux virtual machine on
```

Comment:
@RFMVasconcelos (Canonical) Can you check if there's any overlap with your guide (https://github.com/kubeflow/website/blob/master/content/en/docs/started/workstation/getting-started-multipass.md) - maybe we can combine the content instead of duplicating it here?

@evilnick WDYT?

Reply:
@8bitmp3 thank you for taking the time to review this! In fact it was quite WIP. :) 

I've corrected this with my review, I believe. Less marketing fluff and no duplication. 🚀

Reply:
@8bitmp3 thanks for looking this over